docs/source/deploypeer/peerchecklist.md

```diff
@@ -0,0 +1,399 @@
+
+  - **`pvtData.implicitCollectionDisseminationPolicy.maxPeerCount`:** (Recommended that you override this value when using private data implicit collections.) **New in Fabric 2.0.** This is an organization-specific collection setting that is used to ensure the private data is disseminated elsewhere in case this peer endorses a request and then goes down for some reason. While the `requiredPeerCount` specifies the number of peers that must get the data, the `maxPeerCount` is the number of attempted peer disseminations. The default is set to `1` but in a production environment with `n` peers in an organization, the recommended setting is `n-1`.
+
+## peer.tls.*
+
+```
+tls:
+# Require server-side TLS
+enabled:  false
+# Require client certificates / mutual TLS.
+# Note that clients that are not configured to use a certificate will
+# fail to connect to the peer.
+clientAuthRequired: false
+# X.509 certificate used for TLS server
+cert:
+    file: tls/server.crt
+# Private key used for TLS server (and client if clientAuthEnabled
+# is set to true
+key:
+    file: tls/server.key
+# Trusted root certificate chain for tls.cert
+rootcert:
+    file: tls/ca.crt
+# Set of root certificate authorities used to verify client certificates
+clientRootCAs:
+    files:
+      - tls/ca.crt
+```
+
+Configure this section to enable TLS communications for the peer. After TLS is enabled, all nodes that transact with the peer will also need to enable TLS. Review the topic on [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) for instructions on how to generate the peer TLS certificates.
+
+- **`enabled`:** (Default value should be overridden.) To ensure your production environment is secure, TLS should be enabled for all communications between nodes by setting `enabled: true` in the `tls` section of the config file. While this field is disabled by default, which may be acceptable for a test network, it should to be enabled when in production. This setting will configure **server-side TLS**, meaning that TLS will guarantee the identity of the _server_ to the client and provides a two-way encrypted channel between them.
+
+- **`cert.file`:** (Default value should be overridden.) Every peer needs to register and enroll with its TLS CA before it can transact securely with other nodes in the organization. Therefore, before you can deploy a peer, you must first register a user for the peer and enroll the peer identity with the TLS CA to generate the peer's TLS signed certificate. If you are using the recommended folder structure from the [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) topic, this file needs to be copied into `config/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls`
```

Comment:
config/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls is for test network only.